org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -21,16 +21,46 @@ package lang {
 	 * @author jfernandes
 	 * since 1.0
 	 */
-	class WObject {
+	class Object {
 		method identity() native
+		method instanceVariables() native
```

Comment:
Why do we need this?
I would prefer that wollok objects do not understand this kind of reflective methods and use a mirror instead.